ts/sdk/src/jitProxyClient.ts

```diff
@@ -135,4 +143,41 @@ export class JitProxyClient {
 			.remainingAccounts(remainingAccounts)
 			.instruction();
 	}
+
+	public async getCheckOrderConstraintIx({
+		subAccountId,
+		orderConstraints,
+	}: {
+		subAccountId: number;
+		orderConstraints: OrderConstraint[];
+	}): Promise<TransactionInstruction> {
+		subAccountId =
+			subAccountId !== undefined
+				? subAccountId
+				: this.driftClient.activeSubAccountId;
+
+		const readablePerpMarketIndex = [];
+		const readableSpotMarketIndexes = [];
+		for (const orderConstraint of orderConstraints) {
+			if (isVariant(orderConstraint.marketType, 'perp')) {
+				readablePerpMarketIndex.push(orderConstraint.marketIndex);
```

Comment:
should filter for duplicates?

Reply:
`getRemainingAccounts` will handle duplicate market indexes